Src/Plugins/Security/SecurePlaintextSecretsValidators/SEC101_005.SlackApiKeyValidator.cs

```diff
@@ -44,13 +45,15 @@ protected override IEnumerable<ValidationResult> IsValidStaticHelper(IDictionary
                                                                 ref ResultLevelKind resultLevelKind)
         {
             const string uri = "https://slack.com/api/auth.test";
+            string secret = fingerprint.Secret;
```

Comment:
>secret [](http://example.com/codeflow?start=19&length=6)

note how I continue to make all the code as consistent as possible. it's a big help when making mechanical changes such as this one.